containers/server-image/server-image.changes.deneb-alpha.NoToolsOnContainers

```diff
@@ -0,0 +1 @@
+- Do not install mgrctl and mgradm in the container
```

Comment:
Installing and removing is not exactly the same than not installing.
Maybe we can say here, remove unnecessary installed mgrctl and mgradm files.

Also, do we have the same issue with mgrpxy on the proxy container, right?

Reply:
looks a bit hacky to me.